app/src/main/java/aero/minova/cas/setup/InstallToolIntegration.java

```diff
@@ -60,8 +61,7 @@ public void installSetup(Path setupXml) {
 			BaseSetup.hashModules = new Hashtable<>();
 			BaseSetup.hashtables = new Hashtable<>();
 			BaseSetup.tablevector = new Vector<>();
-			final BaseSetup setup = new BaseSetup();
-			setup.setIsFatJarMode(isFatJarMode);
```

Comment:
Ist hier absichtlich die FATJAR rausgeflogen?

Reply:
ja, das tut in BaseSetup eh nichts, und wir wollen Warnungen soweit wie möglich vermeiden.